packages/runtime-common/indexer.ts

```diff
@@ -499,6 +504,144 @@ export class Indexer {
     ];
   }
 
+  private buildPrerenderedInstancesQuery(everyCondition: CardExpression) {
+    // For evety indexed card instance (type = instance), go through its deps and find the corresponding CSS index entries (type = css)
+    return [
+      `WITH instance_records AS (
+        SELECT i.file_alias, i.deps, i.embedded_html ->> 'default' as default_embedded_html, i.search_doc, i.url, r.realm_url
+        FROM boxel_index AS i ${tableValuedFunctionsPlaceholder}
+        INNER JOIN realm_versions r ON i.realm_url = r.realm_url`,
+      'WHERE',
+      ...everyCondition,
+      `),
+      deps_expanded AS (
+        SELECT
+          ir.file_alias AS instance_file_alias,
+          ir.default_embedded_html AS instance_embedded_html,
+          ir.realm_url,
+          ir.search_doc,
+          ir.url,
+          jsonb_array_elements_text(ir.deps) AS dep
```

Comment:
does this work in sqlite? i would have expected that you did something more similar to how we handle types in the main query....

Reply:
i double-checked--yeah it looks like we replace this with a `json_each` in sqlite which should work...